packages/fuels-test-helpers/src/script.rs

```diff
@@ -0,0 +1,30 @@
+use fuel_core::service::{Config, FuelService};
+use fuel_gql_client::client::FuelClient;
+use fuel_gql_client::fuel_tx::{Receipt, Transaction};
+use fuels_contract::script::Script;
+use std::fs::read;
+
```

Comment:
Please add a docstring to this function so people can understand at a glance what it does and its reason for existing (ie making testing scripts write less boilerplate)

Reply:
Is it only for `boolean` values? 
Remove the `.0` typo